packages/cli/src/commands/integration/add.ts

```diff
@@ -1,116 +1,154 @@
-import { Team } from '@vercel-internals/types';
+import chalk from 'chalk';
 import open from 'open';
 import Client from '../../util/client';
-import getUser from '../../util/get-user';
+import formatTable from '../../util/format-table';
+import { packageName } from '../../util/pkg-name';
+import getScope from '../../util/get-scope';
+import list from '../../util/input/list';
+import cmd from '../../util/output/cmd';
+import indent from '../../util/output/indent';
 import { getLinkedProject } from '../../util/projects/link';
-import getTeamById from '../../util/teams/get-team-by-id';
-
-export interface IntegrationProduct {
-  id: string;
-  slug: string;
-  name: string;
-  shortDescription: string;
-  type: 'storage' | string;
-}
-
-export interface Integration {
-  id: string;
-  slug: string;
-  name: string;
-  products?: IntegrationProduct[];
-}
+import {
+  BillingPlan,
+  Integration,
+  IntegrationInstallation,
+  IntegrationProduct,
+  Metadata,
+} from './types';
+import { createMetadataWizard, MetadataWizard } from './wizard';
+import {
+  fetchIntegration,
+  fetchInstallations,
+  fetchBillingPlans,
+  provisionStoreResource,
+  connectStoreToProject,
+} from './client';
 
 export async function add(client: Client, args: string[]) {
   if (args.length > 1) {
-    client.output.error(`Can't install more than one integration at a time.`);
+    client.output.error('Cannot install more than one integration at a time');
     return 1;
   }
 
   const integrationSlug = args[0];
 
-  const user = await getUser(client);
-  const teamId = client.config.currentTeam ?? user.defaultTeamId;
+  if (!integrationSlug) {
+    client.output.error('One argument is required');
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** Can we make this message easier to understand when output?

```suggestion
    client.output.error(`Error: You must pass an integration slug.`);
```

What do we call the integration identifier publicly?

Reply:
Integration slug is probably the right term for now.